test/exports.test.js

```diff
@@ -6,8 +6,14 @@ import { locales as isAlphanumericLocales } from '../src/lib/isAlphanumeric';
 import { locales as isMobilePhoneLocales } from '../src/lib/isMobilePhone';
 import { locales as isFloatLocales } from '../src/lib/isFloat';
 import { locales as ibanCountryCodes } from '../src/lib/isIBAN';
+import { locales as isPassportNumberLocales } from '../src/lib/isPassportNumber';
```

Comment:
The idea is good. Perhaps just for better clarity, dropping the `is` is better? Since `is` denotes doing a check which should be a function, but this is a list. So just `passportNumberLocales` is good with me.

Reply:
I can do that, but it won't be following the same naming convention as the rest of the imports in the `exports.test.js` (barring the `ibanCountryCodes` outlier).  I went with consistency in naming.

Please let me know if I should change it or leave it as-is.

Reply:
@derekparnell do change . I think we need to drop the is prefix on others as well . Doesnt make sense

Reply:
ok.  I made the change from `isPassportNumberLocales` to `passportNumberLocales`.  I did not change the others, as I understand that would be in another issue ticket, since it's unrelated to this issue (although the theme is similar).

Reply:
yeah . I will do the cleanup in another PR shortly